lib/environment/class-environment.php

```diff
@@ -9,6 +9,20 @@ class Environment {
 	 */
 	const VAR_PREFIX = 'VIP_ENV_VAR';
 
+	/**
+	 * Checks existence of custom environment variable values based on the supplied key and return
+	 * its name if it is set, null otherwise.
+	 *
+	 * @param string $key The name of the environment variable.
+	 * @return string|null
+	 */
+	public static function has_var( string $key ): ?string {
```

Comment:
`has_var()` should only check for the existence of the constant and return a `boolean` value rather than the constant if it's defined or `null` for undefined.